subsys/net/l2/openthread/Kconfig.features

```
@@ -182,3 +182,6 @@ config OPENTHREAD_OTNS
 
 config OPENTHREAD_FULL_LOGS
 	bool "Enable OpenThread full logs"
+
+config OPENTHREAD_CONFIG_MLE_LINK_METRICS_ENABLE
```

```suggestion
config OPENTHREAD_MLE_LINK_METRICS_ENABLE
```
Same way as with other configurations.